spec/requests/api/v1/customers/request_spec.rb

```diff
@@ -145,7 +145,7 @@
   end
 
   it "finds all customers matching created_at timestamps" do
-    create_list(:customer, 3)
+    create_list(:customer, 3, created_at: Time.now)
```

Comment:
We should ask @sallymacnicholas about these strange errors/bugs we were getting.